src/test/java/step33/CarRacingTest.java

```diff
@@ -0,0 +1,39 @@
+package step33;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+import java.util.Random;
+
+
+class CarRacingTest {
+
+    private CarRacing carRacing;
+
+    @BeforeEach
+    void init() {
+        carRacing = new CarRacing(1);
+    }
+
+    @DisplayName("CarRacing 횟수가 0회 일 경우 테스트")
+    @Test
+    void carResponsesIsEmptyTrue() {
+        List<CarResponse> start = carRacing.start(new RandomGenerator(new Random()), 0);
+        Assertions.assertEquals(start.isEmpty(), true);
```

Comment:
```suggestion
        assertTrue(start.isEmpty());
```
assertTrue를 사용해볼 수 있을 것 같아요! 😄